### DISCUSSION TITLE
Option to advance (or delay) audio on OBC.

### DISCUSSION THREAD

**Discussion by JetsonFlyers** [f1multiviewer/issue-tracker#58] [Feature requests]
The actual team radio is delayed around 5 seconds on the OBC streams. You can see this when watching drivers open their mics on the wheel and then not hearing the audio until a few seconds later. When watching with the MVC player I used to advance the audio of those streams to account for this difference so that the audio of the team radio was 'accurate'. It would be nice to have that option with F1MV.

**Comment by stefanverkerknl** [comment]
The problem is (at least for me) that the car sounds (powerunit accelerating etc.) is not delayed. It's only the team radio. So that would mess up the audio of the car.

**Comment by f1multiviewer** [comment]
I did look into this, but unfortunately bitmovin (the player used) doesn't have any option whatsoever to shift the audio relative to the video. The only way would be to create two players, but that would mean double the bandwidth/decode use, since there is also no option to turn off video/audio download/decode/playback.

**Reply by f1multiviewer** [reply]
I've added a `wontfix` for now, but will leave it open to see how much interest there is (given the limitation @stefanverkerknl mentioned is unfixable).
